src/drawers/errorBarDrawer.ts

```diff
@@ -0,0 +1,77 @@
+///<reference path="../reference.ts" />
+
+module Plottable {
+export module _Drawer {
+  export class ErrorBar extends Element {
+    public static ERROR_BAR_CLASS = "error-bar";
+    public static ERROR_BAR_MIDDLE_CLASS = "error-bar-middle";
+    public static ERROR_BAR_LOWER_CLASS = "error-bar-lower";
+    public static ERROR_BAR_UPPER_CLASS = "error-bar-upper";
+
+    private _errorTickLength = 20;
+    private _isVertical: boolean;
+
+    constructor(key: string, isVertical: boolean) {
+      super(key);
+      this.svgElement("g");
```

Comment:
My understanding is that this was responsible for indicating that the containing element that attributes should be applied to was "g"